bin/yaml/v.yaml

```diff
@@ -0,0 +1,11 @@
+compilers:
+  v:
+    type: ziparchive
+    dir: v-{name}
+    url: https://github.com/vlang/v/releases/download/weekly.{name}/v_linux.zip
+    folder: v
+    check_exe: v --version
+    after_stage_script:
+      - export VMODULES=$HOME/.vmodules
+    targets: 
+      - '2023.30'
```

Comment:
Please add a newline at the end of the file.

Reply:
fixed in [d200adb](https://github.com/compiler-explorer/infra/pull/1058/commits/d200adb818420c97bf10a947b8cc984d4c1d58e4)